src/windows/NewInstanceWindow.tsx

```diff
@@ -80,10 +78,10 @@ const NewInstanceWindow = () => {
                     {err}
                 </div>
             )}
-            <h2 className="font-xl font-bold mb-2">New Instance</h2>
+            <h2 className="font-2xl font-bold mb-2">New Instance</h2>
             <form
                 onSubmit={handleFormSubmit}
-                className="flex container flex-col space-y-1"
+                className="flex container flex-col gap-4"
```

Comment:
This gap is probably too large and creates a scrollbar on the window.
```suggestion
                className="flex container flex-col gap-3"
```